### PR TITLE
Add link when used as a widget

### PR DESCRIPTION
**Pull request by leemyongpakvn**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | [function getWidgetVariables](https://github.com/PrestaShop/blockreassurance/blob/dev/blockreassurance.php#L430-L454) does not return block's link, then [function renderWidget](https://github.com/PrestaShop/blockreassurance/blob/dev/blockreassurance.php#L411) does not show block's link when the modules is used as a widget
| Type?         | improvement
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/21752
| How to test? | Here are steps to test with classic-theme: <br>- Apply the PR changes <br>- **Rename** `blockreassurance.tpl` in [classic-theme](https://github.com/PrestaShop/classic-theme/tree/develop/modules/blockreassurance/views/templates/hook) folder to other (`block_hook_reassurance.tpl` e.g) to force using the module's widget template file instead. <br>- Put this widget code `{widget name='blockreassurance'}` somewhere in [category-footer.tpl](https://github.com/PrestaShop/classic-theme/blob/develop/templates/catalog/_partials/category-footer.tpl) (at line 25 e.g)<br>- In BO: Adv. Parameters > Performance > Clear cache to force using new template; set at least a link for a block in module's Config content<br>- In FO: open a category page to see the widget with block's link in the bottom of page. <br>_(Here are steps for a quick test, in fact, you need to copy new code from the module widget template to the theme override template appropriately :)_

### CODE REVIEW

File: views/templates/hook/blockreassurance.tpl [hunk 1]
```diff
@@ -20,7 +20,12 @@
     <div id="block-reassurance">
         <ul>
             {foreach from=$elements item=element}
-                <li><img src="{$element.image}" alt="{$element.text|escape:'quotes'}"/> <span>{$element.text}</span></li>
+                <li>
+                    <div class="block-reassurance-item" {if $element['type_link'] !== $LINK_TYPE_NONE && !empty($element['link'])} style="cursor:pointer;" onclick="window.open('{$element['link']}')"{/if}>
```
Comment (nicosomb) [inline views/templates/hook/blockreassurance.tpl:24]: * You have a CSS class, so you can put the style in this class. 
* I'm not fan of the `onclick` method to emulate a link

**Review by matks — CHANGES_REQUESTED** [review]
Thank you @nicosomb for review I agree with you ;)

File: views/templates/hook/blockreassurance.tpl (hunk 1 above)
Comment (NeOMakinG) [reply views/templates/hook/blockreassurance.tpl:24]: I don't get the reason why people who developed this didn't use links in the past, is there a valid reason?

Comment (leemyongpakvn) [reply views/templates/hook/blockreassurance.tpl:24]: @NeOMakinG Neither do I. By the way, we have 3 template files with `onclick`, so one more won't make this module worse.
Solving the `onclick` in another issue and PR will be safer and make less confuses.

**Review by NeOMakinG — APPROVED** [review]
As the module is mainly using that onclick hack, I would myself accept this PR and make the global refactor another issue/follow-up PR

**Review by atomiix — APPROVED** [review]

**Comment by leemyongpakvn** [comment]
Hello @matks, we are waiting your final decision here :pray:

**Review by matks — APPROVED** [review]
Thank you for ping @leemyongpakvn you're right

I think it's a good thing you opened PR https://github.com/PrestaShop/blockreassurance/pull/455/ so I approve this PR ✅

**Review by florine2623 — APPROVED** [review]
Hello @leemyongpakvn ,

Tested on PS 8.0.0, with PHP 7.4 and blockreassurance 5.1.1.

Here's what I have:

https://user-images.githubusercontent.com/16019289/203086677-0ed7c8eb-2373-42a2-a5fe-cd967236962f.mov

Links work well as a widget.

It is QA ✅

**Comment by okom3pom** [comment]
Thank you @leemyongpakvn && @florine2623

**Pull request merged**